_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -22,28 +18,41 @@ acknowledgements:
 
 ## Applicability
 
-The rule applies to any [keyboard shortcut][] that requires pressing only [printable character][] keys to trigger within a [HTML document][].
+The rule applies to any [keyboard event][]: 
+ - where the attribute `key` is a [printable character][] key; and
+ - the method `getModifierState` returns `false`; and
+ - is [dispatched][] to an [event target][] within a [HTML document][].
 
-## Expectation
+## Expectation 1
 
-For the test target, the outcome of at least one of the following rules is passed:
+For each test target:
+ - an [instrument][] to prevent any result of the [keyboard event][] is available; or
+ - if the [event target][] doesn't have [focus][], any result of the event is prevented; or
+ - an [instrument][] is available to force the [keyboard event][] to have the method `getModifierState` return `true`
 
-- [Printable keys only shortcut can be disabled](https://act-rules.github.io/rules/670a30)
-- [Printable keys only shortcut can be remapped](https://act-rules.github.io/rules/aa8b52) 
-- [Printable key shortcut is only active when component has focus](https://act-rules.github.io/rules/1e9941)
+**Note:** The same [instrument][] can be used to disable/force more than one event.
 
-## Assumptions
+**Note:** Preventing results of the event can be done in multiple ways (e.g. removing the event listener; handling the event in a different manner; ...) but the way in which it is done is not relevant for this rule.
```

Comment:
I also think this can be removed. Especially as we state that this is not relevant for this rule.

Reply:
Same as the comment above, i think it is better to leave the note. This note is also being used in similar rules that target events, like the device motion one.